renderer/locales/id.json

```diff
@@ -0,0 +1,288 @@
+      "BUTTON_FOLDER": "Pilih Folder",
+      "DESCRIPTION": "Anda dapat menambahkan model Anda sendiri dengan mudah. Untuk detail lebih lanjut:",
+      "LINK_TITLE": "Repositori Model Kustom"
+    },
+    "CUSTOM_INPUT_RESOLUTION": {
+      "TITLE": "LEBAR OUTPUT KUSTOM",
+      "RESTART": "MEMBUTUHKAN RESTART",
+      "DESCRIPTION": "Gunakan lebar khusus untuk gambar output. Tinggi akan disesuaikan secara otomatis. Mengaktifkan ini akan menimpa pengaturan skala."
+    },
+    "DONATE": {
+      "DESCRIPTION": "Jika Anda menyukai apa yang kami lakukan :)",
+      "BUTTON_TITLE": "💎 DONASI"
+    },
+    "GPU_ID_INPUT": {
+      "TITLE": "ID GPU",
+      "DESCRIPTION": "Silakan baca Dokumentasi Upscayl untuk informasi lebih lanjut.",
+      "ADDITIONAL_DESCRIPTION": "Aktifkan mode kinerja tinggi di Windows untuk hasil yang lebih baik."
+    },
+    "IMAGE_FORMAT": {
+      "TITLE": "SIMPAN GAMBAR SEBAGAI",
+      "PNG": "PNG",
+      "JPG": "JPG",
+      "WEBP": "WEBP"
+    },
+    "IMAGE_SCALE": {
+      "TITLE": "Skala Gambar",
+      "DESCRIPTION": "Apa pun di atas 4X (kecuali 16X Double Upscayl) hanya mengubah ukuran gambar dan tidak menggunakan peningkatan AI.",
+      "WARNING": "Nilai di atas 5X dapat menyebabkan masalah kinerja pada beberapa perangkat!",
+      "ADDITIONAL_WARNING": "Hal ini dapat menyebabkan masalah kinerja pada beberapa perangkat!"
+    },
+    "LOG_AREA": {
+      "ON_COPY": "DISALIN ✅",
+      "BUTTON_TITLE": "SALIN LOG 📋",
+      "NO_LOGS": "Tidak ada log untuk ditampilkan"
+    },
+    "OVERWRITE_TOGGLE": {
+      "TITLE": "TIMPA UPSCALE SEBELUMNYA",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl akan memproses gambar itu lagi alih-alih memuatnya secara langsung."
+    },
+    "RESET_SETTINGS": {
+      "BUTTON_TITLE": "RESET UPSCAYL",
+      "ALERT": "Upscayl telah direset. Silakan restart aplikasi."
+    },
+    "SAVE_OUTPUT_FOLDER": {
+      "TITLE": "SIMPAN FOLDER OUTPUT",
+      "DESCRIPTION": "Jika diaktifkan, folder output akan diingat antara sesi."
+    },
+    "AUTO_UPDATE": {
+      "TITLE": "PEMBARUAN OTOMATIS UPSCAYL",
+      "DESCRIPTION": "Jika diaktifkan, aplikasi akan memeriksa pembaruan baru dan memberi tahu Anda."
+    },
+    "ENABLE_CONTRIBUTION": {
+      "TITLE": "BANTU TINGKATKAN UPSCAYL",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl akan mengumpulkan data penggunaan anonim untuk meningkatkan antarmuka dan fitur aplikasi."
+    },
+    "THEME": {
+      "TITLE": "TEMA UPSCAYL"
+    },
+    "LANGUAGE": {
+      "TITLE": "BAHASA UPSCAYL"
+    },
+    "CUSTOM_TILE_SIZE": {
+      "TITLE": "UKURAN TILE KUSTOM",
+      "DESCRIPTION": "Gunakan ukuran tile kustom untuk membagi gambar. Ini dapat membantu memproses gambar lebih cepat dengan mengurangi jumlah tile yang dihasilkan."
+    },
+    "TURN_OFF_NOTIFICATIONS": {
+      "TITLE": "MATIKAN NOTIFIKASI",
+      "DESCRIPTION": "Jika diaktifkan, Upscayl tidak akan mengirim pemberitahuan sistem apapun tentang keberhasilan atau kegagalan."
```

Comment:
Suggest: Jika diaktifkan, Upscayl tidak akan mengirim pemberitahuan sistem apapun ketika berhasil ataupun gagal.

Reply:
Thanks! Agreed, I'll make the change.